lib/sonarQube.ts

```diff
@@ -53,6 +53,11 @@ export interface SonarQubeSupportOptions {
      */
     mvnSonarArgs?: string[];
 
+    /**
+     * Extra command arguments to supply to dotnet sonarscanner
+     */
+    dotNetCoreArgs?: string[];
```

Comment:
Not so sure whether I like this being added here, as this doesn't really scale with regards to other stacks, per stack we'll end up adding an args field.

Reply:
@lievendoclo Yea I don't disagree; though I'll admit I'm not sure what a better approach is without making the scanners themselves pluggable somehow.  Any guidance on how to structure this would be appreciated - as we now have this for Maven, dotnet, and the scanner utility itself.

Reply:
@lievendoclo bump